src/js/utils/DOM.js

```diff
@@ -65,6 +65,16 @@ export const containsFocus = (node) => {
   return !!element;
 };
 
+export const focusWithinDropPortal = (node) => {
```

Comment:
small naming thought -- we have `containsFocus` function already so wondering if we name this `portalContainsFocus` or something that aligns with that. or `dropContainsFocus`